spec/lib/email/sender_spec.rb

```diff
@@ -629,7 +631,14 @@ def stub_deliver_response(message)
           expect(message.to_s.scan(/cid:[\w\-@.]+/).uniq.length).to eq(2)
         end
 
-        it "does not attach images that are not marked as secure" do
+        it "does not attach images that are not marked as secure, in the case of a non-secure upload copied to a PM" do
```

Comment:
`CookedPostProcessor` always updates upload security now, so the original way this was tested didn't work. Only in the case of an upload copied from a public post into a PM/secure category would we end up with a mix of secure/non-secure images in a post.